README.md

```diff
@@ -19,113 +19,122 @@ This library includes the following modules.
   - [`Semigroup[T]`](https://typelevel.org/cats/api/cats/kernel/Semigroup.html)
   - [`Monoid[T]`](https://typelevel.org/cats/api/cats/kernel/Monoid.html)
   - [`Group[T]`](https://typelevel.org/cats/api/cats/kernel/Group.html)
-- `magnolify-scalacheck` - type class derivation for [ScalaCheck](https://github.com/typelevel/scalacheck) 
+- `magnolify-scalacheck` - type class derivation for [ScalaCheck](https://github.com/typelevel/scalacheck)
   - [`Arbitrary[T]`](https://github.com/typelevel/scalacheck/blob/master/doc/UserGuide.md#universally-quantified-properties)
-  - [`Cogen[T]`](https://github.com/typelevel/scalacheck/blob/master/src/main/scala/org/scalacheck/Cogen.scala#L20-L29)
+  - [`Cogen[T]`](https://github.com/typelevel/scalacheck/blob/master/src/main/scala/org/scalacheck/Cogen.scala)
 - `magnolify-guava` - type class derivation for [Guava](https://guava.dev)
-  - [`Funnel[T]`](https://guava.dev/releases/25.0-jre/api/docs/com/google/common/hash/Funnel.html)
-  
-- `magnolify-avro` - conversion between Scala case classes and [Apache Avro](https://github.com/apache/avro) `GenericRecord`
-- `magnolify-bigquery` - conversion between Scala case classes and [Google Cloud BigQuery](https://cloud.google.com/bigquery/) `TableRow`
-- `magnolify-datastore` - conversion between Scala case classes and [Google Cloud Datastore](https://cloud.google.com/datastore/) `Entity`
-- `magnolify-tensorflow` - conversion between Scala case classes and [TensorFlow](https://www.tensorflow.org/) `Example`
+  - [`Funnel[T]`](https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/hash/Funnel.html)
+
+- `magnolify-avro` - conversion between Scala types and [Apache Avro](https://github.com/apache/avro) `GenericRecord`
+- `magnolify-bigquery` - conversion between Scala types and [Google Cloud BigQuery](https://cloud.google.com/bigquery/) `TableRow`
+- `magnolify-datastore` - conversion between Scala types and [Google Cloud Datastore](https://cloud.google.com/datastore/) `Entity`
+- `magnolify-tensorflow` - conversion between Scala types and [TensorFlow](https://www.tensorflow.org/) `Example`
 
 # Usage
 
 Cats, ScalaCheck, and Guava type class derivation can be performed both automatically and semi-automatically.
 
-Automatic derivation with implicits:
```

Comment:
Honestly don't know how to explain auto vs semiauto here since it requires some experience with both to see the pros & cons. So not sure if we should event attempt to in README that's supposed to be concise.